l10n_it_withholding_tax/models/account.py

```diff
@@ -592,6 +592,33 @@ class AccountMoveLine(models.Model):
     withholding_tax_generated_by_move_id = fields.Many2one(
         "account.move", string="Withholding Tax generated from", readonly=True
     )
+    withholding_tax_amount_net_pay = fields.Float(
+        "Withholding tax amount net pay",
+        compute="_compute_withholding_tax_amount_net_pay",
+    )
+    withholding_tax_amount_net_pay_residual = fields.Float(
+        "Withholding tax amount net pay residual",
+        compute="_compute_withholding_tax_amount_net_pay",
+    )
+
+    def _compute_withholding_tax_amount_net_pay(self):
+        for record in self:
```

Comment:
Secondo me sarebbe più leggibile
```suggestion
        for withholding_tax in self:
```
o `wt_tax`, o comunque qualcosa che faccia capire cosa c'è dentro questa variabile, cosa ne pensi?